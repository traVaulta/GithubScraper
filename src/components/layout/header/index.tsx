import React from 'react';
import SponsorImg from '../../../assets/powered-by.png';
import LogoImg from '../../../assets/app-logo.png';

const AppHeader = () => (
    <header className="app-header app-header--outline">
        <div className="app-header app-header--sponsor">
            <img src={LogoImg} alt="Logo"/>
            <img src={SponsorImg} alt="powered-by-rimac"/>
        </div>
        <div>Github Scraper</div>
    </header>
);

const Header = React.memo(AppHeader);

export default Header;
