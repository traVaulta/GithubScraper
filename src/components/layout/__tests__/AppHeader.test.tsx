import React from 'react';
import {render} from '@testing-library/react';

import {AppHeader} from '../header';

describe(`${AppHeader.name}`, () => {
    it('should render', () => {
        const {asFragment} = render(<AppHeader/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
