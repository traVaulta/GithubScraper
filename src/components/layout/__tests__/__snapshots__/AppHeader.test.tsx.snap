// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should render 1`] = `
<DocumentFragment>
  <header
    class="app-header app-header--outline"
  >
    <div
      class="app-header app-header--sponsor"
    >
      <img
        alt="Logo"
        src="app-logo.png"
      />
      <img
        alt="powered-by-rimac"
        src="powered-by.png"
      />
    </div>
    <div>
      Github Scraper
    </div>
  </header>
</DocumentFragment>
`;
